frontend/src/components/RoundResultScreen.js

```diff
@@ -13,47 +13,77 @@ const RoundResultsScreen = ({
   currentUserPlayerId,
   timer,
   correctAnswer,
+  powerupStatus,
 }) => {
-  let correctSound = new Audio('https://vgmsite.com/soundtracks/nintendo-switch-sound-effects/zfoeyezf/Jig%200.mp3');
-  let incorrectSound = new Audio('https://vgmsite.com/soundtracks/nintendo-switch-sound-effects/lzpzngfl/Jig%201.mp3');
+  const activatePowerup = () => {
```

Comment:
instead of creating a function here that makes the socket emit the event, we could do something similar to the QuestionScreen selectAnswer logic, where we pass `onSelectAnswer` (we would do a `onActivatePowerup`

this is because we don't have the `socket` instance in this screen to do the `.emit()`